src/video.js

```diff
@@ -1,11 +1,52 @@
-import {deepAccess, logError} from './utils.js';
+import {deepAccess, isArrayOfNums, isInteger, isNumber, isStr, logError, logWarn} from './utils.js';
 import {config} from '../src/config.js';
 import {hook} from './hook.js';
 import {auctionManager} from './auctionManager.js';
 
 export const OUTSTREAM = 'outstream';
 export const INSTREAM = 'instream';
 
+/**
+ * Basic validation of OpenRTB 2.x video object properties.
+ * Not included: `companionad`, `durfloors`, `ext`
+ * reference: https://github.com/InteractiveAdvertisingBureau/openrtb2.x/blob/main/2.6.md
+ */
+export const ORTB_VIDEO_PARAMS = new Map([
```

Comment:
While this feels like a lot to add to core, so many prominent bid adapters do exactly this that it works likely be a net code reduction